step-06/lib/shopping_cart_tab.dart

```diff
@@ -1,334 +1 @@
-    return _ShoppingCartTabState();
-  }
-}
-
-class _ShoppingCartTabState extends State<ShoppingCartTab> {
-  String name;
-  String email;
-  String location;
-  String pin;
-  DateTime dateTime = DateTime.now();
-  final _currencyFormat = NumberFormat.currency(symbol: '\$');
-
-  Widget _buildNameField() {
-    return CupertinoTextField(
-      prefix: const Icon(
-        CupertinoIcons.person_solid,
-        color: CupertinoColors.lightBackgroundGray,
-        size: 28,
-      ),
-      padding: const EdgeInsets.symmetric(horizontal: 6, vertical: 12),
-      clearButtonMode: OverlayVisibilityMode.editing,
-      textCapitalization: TextCapitalization.words,
-      autocorrect: false,
-      decoration: const BoxDecoration(
-        border: Border(
-          bottom: BorderSide(
-            width: 0,
-            color: CupertinoColors.inactiveGray,
-          ),
-        ),
-      ),
-      placeholder: 'Name',
-      onChanged: (newName) {
-        setState(() {
-          name = newName;
-        });
-      },
-    );
-  }
-
-  Widget _buildEmailField() {
-    return const CupertinoTextField(
-      prefix: Icon(
-        CupertinoIcons.mail_solid,
-        color: CupertinoColors.lightBackgroundGray,
-        size: 28,
-      ),
-      padding: EdgeInsets.symmetric(horizontal: 6, vertical: 12),
-      clearButtonMode: OverlayVisibilityMode.editing,
-      keyboardType: TextInputType.emailAddress,
-      autocorrect: false,
-      decoration: BoxDecoration(
-        border: Border(
-          bottom: BorderSide(
-            width: 0,
-            color: CupertinoColors.inactiveGray,
-          ),
-        ),
-      ),
-      placeholder: 'Email',
-    );
-  }
-
-  Widget _buildLocationField() {
-    return const CupertinoTextField(
-      prefix: Icon(
-        CupertinoIcons.location_solid,
-        color: CupertinoColors.lightBackgroundGray,
-        size: 28,
-      ),
-      padding: EdgeInsets.symmetric(horizontal: 6, vertical: 12),
-      clearButtonMode: OverlayVisibilityMode.editing,
-      textCapitalization: TextCapitalization.words,
-      decoration: BoxDecoration(
-        border: Border(
-          bottom: BorderSide(
-            width: 0,
-            color: CupertinoColors.inactiveGray,
-          ),
-        ),
-      ),
-      placeholder: 'Location',
-    );
-  }
-
-  Widget _buildDateAndTimePicker(BuildContext context) {
-    return Column(
-      children: <Widget>[
-        Row(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          children: <Widget>[
-            Row(
-              mainAxisAlignment: MainAxisAlignment.start,
-              children: const <Widget>[
-                Icon(
-                  CupertinoIcons.clock,
-                  color: CupertinoColors.lightBackgroundGray,
-                  size: 28,
-                ),
-                SizedBox(width: 6),
-                Text(
-                  'Delivery time',
-                  style: TextStyle(
```

Comment:
Be consistent with declaring styles in the styles file vs inline

Reply:
Fixed in https://github.com/googlecodelabs/flutter-cupertino-store/pull/6/files#diff-08987380babd0f4baf7e95a853929d94